db/migrate/20170511120047_remove_container_group_failed_sync_event.rb

```diff
@@ -0,0 +1,14 @@
+class RemoveContainerGroupFailedSyncEvent < ActiveRecord::Migration[5.0]
+  class Relationship < ActiveRecord::Base; end
+
+  class MiqEventDefinition < ActiveRecord::Base; end
+
+  def up
+    say_with_time("Remove event definition containergroup_failedsync") do
+      MiqEventDefinition.where(:name => 'containergroup_failedsync').each do |eventdef|
+        Relationship.where(:resource_type => 'MiqEventDefinition', :resource_id => eventdef.id).delete_all
```

Comment:
Yes, it will remove the option to assign the event to policies and remove the event from policies that already have it.
I'm not sure what exactly Relationship covers, but I see that each event has relationship associated with it, which isn't deleted when I delete the event itself. I also saw [here](https://github.com/ManageIQ/manageiq/blob/master/db/migrate/20151222152212_remove_vm_discover_row_from_miq_event_definitions.rb#L9) an example of removing another event and its relationships.